oat-sa/extension-tao-itemqti
# fix: add check of ckeditor existence in gaptext

**Related to**

[AUT-27](https://oat-sa.atlassian.net/browse/AUT-27)

**How to test**

1. Create Item
2. Add Gap Match Interaction
3. Remove it
4. Check console
